### ISSUE TITLE
Does not build with GHC 8.0.2

### ISSUE THREAD

**Issue by 17dec** [JakeWheat/hssqlppp#24]
Trying to build with the Stack lts-8.16 release fails with the following error:

```
hssqlppp-0.6.1: build (lib + test)
Log files have been written to: /home/ayo/hs/hssqlppp/.stack-work/logs/
Progress: 1/4
--  While building package hssqlppp-0.6.1 using:
      /home/ayo/.stack/setup-exe-cache/x86_64-linux/Cabal-simple_mPHDZzAJ_1.24.2.0_ghc-8.0.2 --builddir=.stack-work/dist/x86_64-linux/Cabal-1.24.2.0 build lib:hssqlppp test:Tests --ghc-options " -ddump-hi -ddump-to-file"
    Process exited with code: ExitFailure 1
    Logs have been written to: /home/ayo/hs/hssqlppp/.stack-work/logs/hssqlppp-0.6.1.log
    
    Configuring hssqlppp-0.6.1...
    Preprocessing library hssqlppp-0.6.1...
    Preprocessing test suite 'Tests' for hssqlppp-0.6.1...
    [ 1 of 84] Compiling Database.HsSqlPpp.Utils.Here ( tests/Database/HsSqlPpp/Utils/Here.lhs, .stack-work/dist/x86_64-linux/Cabal-1.24.2.0/build/Tests/Tests-tmp/Database/HsSqlPpp/Utils/Here.o )
    [ 2 of 84] Compiling Database.HsSqlPpp.Utils.GroomUtils ( tests/Database/HsSqlPpp/Utils/GroomUtils.lhs, .stack-work/dist/x86_64-linux/Cabal-1.24.2.0/build/Tests/Tests-tmp/Database/HsSqlPpp/Utils/GroomUtils.o )
          
    /home/ayo/hs/hssqlppp/hssqlppp/tests/Database/HsSqlPpp/Utils/GroomUtils.lhs:22:24: error:
        • Expecting one more argument to ‘Exp’
          Expected a type, but ‘Exp’ has kind ‘* -> *’
        • In the type signature:
            groomF :: Show a => (Exp -> Exp) -> a -> String
          
    /home/ayo/hs/hssqlppp/hssqlppp/tests/Database/HsSqlPpp/Utils/GroomUtils.lhs:22:31: error:
        • Expecting one more argument to ‘Exp’
          Expected a type, but ‘Exp’ has kind ‘* -> *’
        • In the type signature:
            groomF :: Show a => (Exp -> Exp) -> a -> String
make: *** [Makefile:40: test] Error 1

```

**Comment by JakeWheat** [comment]
Thanks for the report, I uploaded a fixed version, 0.6.1. I think it was an issue with an updated version of haskell-src-exts.

**Comment by 17dec** [comment]
Latest version builds fine. Thanks.
